src/assets/i18n/en.json

```diff
@@ -28,7 +28,7 @@
 	"Select a config.map.json file": "Select a config.map.json file",
 	"Importing dashboard...": "Importing dashboard...",
 	"Available dashboards": "Available dashboards",
-	"Name": "Name",
+	"Name": "The dashboard's name",
```

Comment:
Name is used in other places in the builder, it was fixed in the previous PR

Reply:
Rolled back